docs/src/app/components/LeftNav/ExampleOpenRight.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import LeftNav from 'material-ui/lib/left-nav';
+import AppBar from 'material-ui/lib/app-bar';
+import RaisedButton from 'material-ui/lib/raised-button';
+
+export default class LeftNavOpenRightExample extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {open: false};
+  }
+
+  handleToggle = () => this.setState({open: !this.state.open});
+
+  render() {
+    return (
+      <div>
+        <RaisedButton
+          label="Controlled LeftNav That Opens From Right"
+          onTouchTap={this.handleToggle} />
+        <LeftNav width={450} openRight={true} open={this.state.open} >
```

Comment:
how would we handle the number returned by [this](https://github.com/callemall/material-ui/pull/2507/files#diff-c6cf5eece3f28b3dabff47cd1be7cac1L315)